### ISSUE TITLE
`Puppet: Resource` command does not match the output from 'puppet resource ..'

### ISSUE THREAD

**Issue by glennsarti** [jpogran/puppet-vscode#54]
doing a Puppet: Resource command generates the following for a group:
```puppet
group { 'Administrators':
  ensure               => 'present',
  attribute_membership => 'minimum',
  gid                  => 'S-1-5-32-544',
  loglevel             => 'notice',
  members              => ['Administrator', 'glenn.sarti'],
  provider             => 'windows_adsi',
}
```

This does not match the output from `puppet resource group`

I suspect the language server is pulling the puppet resource information too early in the code path

**Comment by glennsarti** [comment]
Fixed in PR #55

**Comment by glennsarti** [comment]
Resolved
